packaging/pom.xml

```
@@ -290,7 +290,7 @@
           <dependency>
             <groupId>com.thoughtworks.xstream</groupId>
             <artifactId>xstream</artifactId>
-          <version>1.4.9</version>
+          <version>1.4.10-java7</version>
```

```suggestion
          <version>1.4.12</version>
```
Let's try 1.4.12 instead. It looks like it works in local builds.